src/transactions/selectors.ts

```diff
@@ -22,6 +23,24 @@ const standbyTransactionsSelector = createSelector(
   }
 )
 
+export const formattedStandByTransactionsSelector = createSelector(
```

Comment:
I didn't really know how to call this, considering there are already selectors like:
- `standByTransactionsSelector`
- `allStandByTransactionsSelector`
- `pendingStandByTransactionsSelector`
- `confirmedStandByTransactionsSelector`

The last three are un-memoized and trigger a lot of re-renders so cannot use them.

Reply:
what about adding this logic to `standByTransactionsSelector`? it seems harmless to ensure the correct types are returned?

Reply:
@kathaypacific `standByTransactionsSelector` includes `getSupportedNetworkIdsForApprovalTxsInHomefeed` which triggers a lot of unnecessary re-renders due to it being unmemoized. This is the reason I didn't use it in the first place and initially used `allStandbyTransactionsSelector` directly.

Reply:
unless it causes noticeable performance degradations, my personal preference would be to use existing functions rather than introduce extra complexity (and potential readability issues). we can go back and improve the memoization for the `getSupportedNetworkIds*` selectors later and fix the issue

Reply:
@kathaypacific my expectation was that once we remove the old feed - we'll adjust all the naming accordingly so there won't be an issue with multiple selectors having similar names and doing similar things.